src/main/java/mate/academy/exception/AuthenticationException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.exception;
+
+public class AuthenticationException extends RuntimeException {
```

Comment:
The `AuthenticationException` should not extend `RuntimeException`. Consider extending a more appropriate exception type, such as `Exception`, to align with the task requirements.